pixeltable/tests/conftest.py

```diff
@@ -15,19 +15,29 @@
 from pixeltable import exprs
 from pixeltable.exprs import RELATIVE_PATH_ROOT as R
 from pixeltable import functions as ptf
-
+import tempfile
+import pathlib
 
 @pytest.fixture(scope='session')
 def init_env(tmp_path_factory) -> None:
     from pixeltable.env import Env
     # set the relevant env vars for Client() to connect to the test db
     home_dir = str(tmp_path_factory.mktemp('base') / '.pixeltable')
+
+    # NB: unix socket path names cannot exceed ~100 chars, 
+    # but pgdata path from pytest fixture exceeds that length.
+    # Will make a separate dir at a shallower path for testing purposes.
```

Comment:
this seems like a general problem. is there no good solution for that? the user may well have a pixeltable home directory that will exceed this threshold.